src/main/java/io/confluent/examples/streams/microservices/OrderDetailsService.java

```diff
@@ -66,23 +67,31 @@ private void startService(String bootstrapServers) {
     try {
       Map<TopicPartition, OffsetAndMetadata> consumedOffsets = new HashMap<>();
       consumer.subscribe(singletonList(Topics.ORDERS.name()));
-      producer.initTransactions();
+      // Commenting out configuration for Exactly Once Semantics (EOS)
+      // because currently EOS is not supported with Confluent Monitoring Interceptors 
+      //producer.initTransactions();
```

Comment:
I am wondering, if we should use `if (eosEnabled)` and have one `public static` variable to enable/disable EOS instead?